src/StSDK.ts

```diff
@@ -35,38 +48,112 @@ export default class StSDK extends BaseSDK {
 		return spread(places, markerSizes, bounds, canvas);
 	}
 
+	/**
+	 * @experimental
+	 */
+	public getRoutesForTripDay(tripId: string, dayIndex: number): Promise<Route[]> {
+		return getRoutesForTripDay(tripId, dayIndex);
+	}
+
+	/**
+	 * @experimental
+	 */
 	public getTrips(dateFrom: string, dateTo: string): Promise<Trip[]> {
 		return getTrips(dateFrom, dateTo);
 	}
 
+	/**
+	 * @experimental
+	 */
 	public getTripDetailed(id: string): Promise<Trip> {
 		return getTripDetailed(id);
 	}
 
+	/**
+	 * @experimental
+	 */
 	public updateTrip(id, dataToUpdate: TripUpdateData): Promise<Trip> {
 		return updateTrip(id, dataToUpdate);
 	}
 
+	/**
+	 * @experimental
+	 */
 	public addPlaceToFavorites(id: string): Promise<void> {
 		return addPlaceToFavorites(id);
 	}
 
+	/**
+	 * @experimental
+	 */
 	public addCustomPlaceToFavorites(name: string, location: Location, address: string): Promise<string> {
 		return addCustomPlaceToFavorites(name, location, address);
 	}
 
+	/**
+	 * @experimental
+	 */
 	public getFavoritesIds(): Promise<string[]> {
 		return getFavoritesIds();
 	}
 
+	/**
+	 * @experimental
+	 */
 	public removePlaceFromFavorites(id: string): Promise<void> {
 		return removePlaceFromFavorites(id);
 	}
 
 	/**
 	 * @experimental
 	 */
-	public getRoutesForTripDay(tripId: string, dayIndex: number): Promise<Route[]> {
-		return getRoutesForTripDay(tripId, dayIndex);
+	public addDayToTrip(id: string): Promise<Trip> {
+		return addDay(id);
+	}
+
+	/**
+	 * @experimental
+	 */
+	public addDayToBeginning(id: string): Promise<Trip> {
```

Comment:
I think that better name is `prependDayToTrip` which is more consistent with `addDayToTrip`